src/viadot/sources/smb.py

```diff
@@ -206,7 +217,7 @@ def _parse_dates(
 
     def _scan_directory(
```

Comment:
Either change this to `_scan_directories` or just call `_scan_directory()` for each path

Reply:
[9e9417e](https://github.com/dyvenia/viadot/pull/1225/commits/9e9417eb3fa10584b698bdfae20aa5ef48698c21)